app/src/main/java/edu/uc/groupproject/covid19tracker/dto/GlobalData.kt

```diff
@@ -12,6 +12,8 @@ data class GlobalData(@SerializedName("total_cases") var totalCases: String,
                       @SerializedName("total_cases_per_1m_population") var totalCasesPer1mPopulation: String,
                       @SerializedName("deaths_per_1m_population") var deathPer1mPopulation: String,
                       @SerializedName("statistic_taken_at") var statisticTakenAt: String) {
+    constructor() : this("", "", "", "", "", "", "", "", "", ""){
```

Comment:
Added secondary constructor so you can make an empty version of the DTO.  Used in a different section for error handling.

Reply:
@mikelawson52 Is this a best practice in kotlin? I don't know why would we need it.

Reply:
Not a best practice, and there are better alternatives. I only used it as a quick way to show some error handling for the null reference pointer exceptions the class could throw on the try catch I added.

Reply:
Another constructor can be clutter, especially since this is just defaulting everything to empty string.  If you'd like another constructor, what I recommend: use one constructor, and provide default values for each of the parameters.  Then, you only need to pass in the values that you have.

With this constructor never actually being invoked, it's clutter, which is technical debt, so I would not merge this as it is now.